### ISSUE TITLE
Planning

### ISSUE THREAD

**Issue by gnarlinsky** [py-cu/post-to-all-the-things#1]
Framework, initial requirements, workflow, etc. etc.

**Comment by jakekreider** [comment]
Really high-level, but is the below sitemap/nav anything like what you had in mind? 

#### Page Layout

```
  Home (displays recent messages) 
  |-> Sign in
  |-> Messages (paginated view of all messages)
      |-> Create Message
          |-> Confirm (and select channels where to publish?)
  |-> Admin
      |-> User admin
      |-> Channel Admin (list avl channels, maintain API keys, disable)
```

**Comment by gnarlinsky** [comment]
Yes!

**Comment by jakekreider** [comment]
Cool, I just pushed a couple of really basic mocks for messages & new message in the `ui-work` branch (just to help get my head clear on what to work towards). Might make one or two more mocks (confirm, maybe sign in), but then start wiring up to an actual application backend.

**Comment by gnarlinsky** [comment]
Cool!  I just pushed a couple tweaks in c08f4b2495f23d8adb1e504a12fbcc125540fb59. 

Also, am I missing something with the project structure of your mocks? Was everything literally in root before you pushed?  I had to de-absolute the img/css/js links to see everything, so I'm wondering what kind of black magic is happening on your end and if I screwed something up by changing the links. :)

**Comment by jakekreider** [comment]
Nope, all good. I was just developing out of that directory for now & using `python -m http.server` from there to host since that'll get blown away when actual rendered pages exist.
